_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,25 @@
+---
+question: "What is a cEP? Why is it so important to do a pre-GSoC analysis
+and design a proposal as early as possible?"
+---
+A **cEP (coala Enhancement Proposal)** is proposed to the **coala** community
+to discuss the new *major features* that the contributor intends to implement
+during the GSoC duration. The proposal is designed before the implementation of
+*major features* because the implementation usually happens from a technical
+perspective which may lead to a poorly designed content, both developer-wise
+and user-wise. So, to avoid this, a cEP should be designed at an earlier stage
+and this essentially saves a lot of coding work and makes sure that major
+features are properly documented.
+
+Experience tells us that the best GSoC students are the ones who heavily
+contribute to the project ideas' website, create projects from the issues, and
+expand on the existing ***.md*** files.
+
+So, if you are planning to participate in GSoC with coala, you should start
+writing a detailed project idea in our [projects](https://github.com/coala/projects)
+repository. Also, you must do enough research about the project in which you
```

Comment:
There are a lot of ```you```s in this sentence. Need to simplify this.

Rephrase it to, ```Also, enough research about the project you are interested in must be done, so...```

Reply:
If sentence contains lot of you(s) we can change it to something like this `Also, do some enough research ... `
I tried to read the sentence with your suggestion I wasn't able to understand line conveys and though of that this sentence needs to be changed.

Have any other suggestion ?

Reply:
Let's see what other reviewers have to say about this.

Reply:
@shashank-b can you please help us here

Reply:
Sure @KVGarg. `Also, do some enough research ...` would not be grammatically correct. Although the current sentences are grammatically correct, changing them to passive voice, as suggested by frextrite, might look formal and nice. Something like `Also, enough research about the project of interest must be done, so...` can be used. But that is definitely a matter of choice. Checking out other such files to see the kind of language that is used might be a nice idea.